sui_core/src/authority/authority_store.rs

```diff
@@ -268,10 +277,10 @@ impl<
     }
 
     /// A function that acquires all locks associated with the objects (in order to avoid deadlocks).
-    async fn acquire_locks(
-        &self,
-        input_objects: &[ObjectRef],
-    ) -> Vec<tokio::sync::MutexGuard<'_, ()>> {
+    async fn acquire_locks<'a, 'b>(
+        &'a self,
+        input_objects: &'b [ObjectRef],
```

Comment:
Was this causing a compile error?  Curious why its needed

Reply:
Yes this was suddenly causing a compiler error with my change. Not sure why